lizzy/version.py

```diff
@@ -1,2 +1,2 @@
 # Version to be checked on the client
-VERSION = '2016-02-22'
+VERSION = '1.4.2'
```

Comment:
this version shouldn't contain the revision, otherwise we'll have to update the client every time we fix a server side error.